Remove test for bundled Qt which breaks dependent applications.  This has
been fixed in 5.13.

Taken from Arch Linux:
https://git.archlinux.org/svntogit/packages.git/tree/trunk/python2-pyqt5-crash-fix.patch?h=packages/pyqt5&id=3e56e11d1fd7b1eac8242ce64c58db2bd9acba20

diff -ur PyQt5_gpl-5.12.3/qpy/QtCore/qpycore_post_init.cpp.in PyQt5_gpl-5.12.3b/qpy/QtCore/qpycore_post_init.cpp.in
--- PyQt5_gpl-5.12.3/qpy/QtCore/qpycore_post_init.cpp.in        2019-06-25 14:41:02.000000000 +0200
+++ PyQt5_gpl-5.12.3b/qpy/QtCore/qpycore_post_init.cpp.in       2019-07-01 17:06:34.882644535 +0200
@@ -151,8 +151,4 @@
     // initialised first (at least for Windows) and this is the only way to
     // guarantee things are done in the right order.
     PyQtSlotProxy::mutex = new QMutex(QMutex::Recursive);
-
-    // Load the embedded qt.conf file if there is a bundled copy of Qt.
-    if (!qpycore_qt_conf())
-        Py_FatalError("PyQt5.QtCore: Unable to embed qt.conf");
 }

